src/platforms/f072/platform.c

```diff
@@ -121,6 +122,11 @@ void platform_request_boot(void)
 
 #pragma GCC diagnostic pop
 
+int platform_hwversion(void)
```

Comment:
Is there a particular reason for removing this from the platform header? This will impair the optimiser's ability to optimise this function out to just a value in all call sites

Reply:
For `f072` the only call site is `cmd_version()` which is cold. This diff hunk is +12 `.text` bytes for that platform, and nothing is spilled on the stack.
`(F072-IF)` does not have a `TRST_PIN` defined, but if added, then same reasoning as for `blackpill-f4` applies.